tox.ini

```diff
@@ -14,7 +14,9 @@ commands=py.test {posargs:tests}
 passenv=
     PYTEST_ADDOPTS
 deps =
-    -r{toxinidir}/requirements-dev.txt
```

Comment:
I tried to remove this `requirement` and added dependency under `deps` of `tox`. I tried lots of options here but failed so this is simplest option I got. @carver calling you for wisdom!

Reply:
Use `extras=test` and you can drop all these `deps`

Reply:
yup, worked like a charm, thanks @carver ! :100: